javarosa/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -90,11 +90,6 @@
     // depend on any result from evaluating tB
     private Vector<Triggerable> triggerables;
 
-    // true if triggerables has been ordered topologically (DON'T DELETE ME
-    // EVEN THOUGH I'M UNUSED)
```

Comment:
Or we can just assume that if it hasn't been used in the last 7 years it won't be used in the next few